### PR TITLE
Adopt arcade.packaging2

### PR DESCRIPTION
**Pull request by ericstj**
Packaging prototype

### CODE REVIEW

**Comment by ericstj** [comment]
FYI: the unneeded rename.
https://github.com/ericstj/wcf/commit/bb7a8b37eab9b9091af837d764ad45068341cf58

**Comment by ericstj** [comment]
@StephenBonikowsky here's the prototype we worked on.

/cc @tmat

File: Directory.Build.props [hunk 1]
```diff
@@ -12,7 +12,7 @@
   <Import Project="eng\FacadeAssemblies.props" />
   <Import Project="eng\Versioning.props"/>
   <Import Project="eng\TestProperties.props"/>
-  <Import Project="eng\uwp.props" />
+  <Import Project="eng\targetFramework.props" />
```
Comment (tmat) [inline Directory.Build.props]: >t [](start = 23, length = 1)

Nit: capitalize file names to make them consistent.

**Comment by tmat** [comment]
This looks very clean. Does it do all we need?

**Comment by StephenBonikowsky** [comment]
It does, we built locally and the packages look great, we did clean-up a few things we didn't need. Now I just need to do the same thing for the other 4 contracts.

File: Directory.Build.props (hunk 1 above)
Comment (StephenBonikowsky) [reply Directory.Build.props]: Will do.

**Comment by tmat** [comment]
Perfect.

File: eng/harvest.targets [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+             Targets="GetPathToHarvest"
+             Properties="PackageToHarvest=$(PackageId)">
+      <Output TaskParameter="TargetOutputs" PropertyName="_pathToHarvest"/>
+    </MSBuild>
+  </Target>
+
+  <Target Name="AddHarvestedFilesToPackage"
+          DependsOnTargets="_GetPathToHarvest">
+    <ItemGroup>
+      <_harvestedItems Include="$(_pathToHarvest)\lib\$(TargetFramework)\*.*" PackagePath="lib\$(TargetFramework)" />
```
Comment (tmat) [inline eng/harvest.targets]: Nit: We generally use `_HarvestedItems` capitalization for "private" properties/items/targets.

**Pull request merged**